pkg/rpc/server/server.go

```diff
@@ -566,9 +566,13 @@ func (s *Server) getClaimable(ps request.Params) (interface{}, *response.Error)
 }
 
 func (s *Server) getNEP5Balances(ps request.Params) (interface{}, *response.Error) {
-	u, err := ps.ValueWithType(0, request.StringT).GetUint160FromHex()
+	val := ps.ValueWithType(0, request.StringT)
+	u, err := val.GetUint160FromHex()
```

Comment:
Maybe we need a `GetUint160FromAddresOrHex`?

Reply:
Done.